src/Raven.Client/Documents/Session/IRevisionsSessionOperations.cs

```diff
@@ -10,6 +10,20 @@
 
 namespace Raven.Client.Documents.Session
 {
+    
+    public enum ForceRevisionStrategy
+    {
+        // Don't force revision
+        None,
+        
+        // Create a forced revision from the document that is currently in store - BEFORE applying any changes made by the user 
+        // The only exception is: with current implementation the forced revision for *new* document only is 'after', not 'before'  
+        Before,
+        
+        // After is currently Not implemented on the server side ! TBD only if requested by customer
+        After
```

Comment:
It can be removed since this isn't implemented yet.

Reply:
done